frontend/src/app/specification-overview/specification-overview.component.ts

```diff
@@ -1,8 +1,10 @@
 import {Component, OnInit} from '@angular/core';
 import {SpecificationService} from '../specification.service';
 import {Specification} from '../models/specification';
+import {Version} from '../models/version';
 import {VersionService} from '../version.service';
 
+/* tslint:disable:member-ordering */
```

Comment:
Why is this needed? Disabling the linter always looks like you're doing it wrong ;)